autoemulate/simulations/base.py

```diff
@@ -151,8 +162,43 @@ def sample_inputs(
         """
         if random_seed is not None:
             set_random_seed(random_seed)  # type: ignore PGH003
-        lhd = LatinHypercube(self.param_bounds)
-        return lhd.sample(n_samples)
+
+        if len(self.sample_param_bounds) == 0:
+            # All parameters are constant - broadcast to n_samples
+            const_vals = torch.tensor(list(self.constant_params.values()))
+            return const_vals.repeat(n_samples, 1)
+
+        if method == "lhs":
```

Comment:
Possibly worth supporting upper and lower case?
```suggestion
        if method.lower() == "lhs":
```